homeassistant/components/gandi_livedns/gandi.py

```diff
@@ -0,0 +1,182 @@
+"""Provides the Gandi.net live DNS access API."""
+import asyncio
+
+import aiohttp
+from aiohttp.hdrs import AUTHORIZATION
+import async_timeout
+
+from homeassistant.const import (
+    CONF_API_KEY,
+    CONF_DOMAIN,
+    CONF_NAME,
+    CONF_TIMEOUT,
+    CONF_TTL,
+    CONF_TYPE,
+)
+from homeassistant.core import HomeAssistant
+
+from .const import (
+    CONF_IPV6,
+    GANDI_LIVEDNS_API_URL,
+    IPV4_PROVIDER_URL,
+    IPV6_PROVIDER_URL,
+)
+
+
+class GandiApiLiveDNS:
```

Comment:
This is code that should be wrapped in a library hosted on PyPi.

See: <https://developers.home-assistant.io/docs/development_checklist>

Reply:
I have make a python package on PyPi